packages/asm/src.ts/assembler.ts

```diff
@@ -1277,8 +1277,8 @@ type _Location = {
 export function parse(code: string, options?: ParserOptions): Node {
     if (options == null) { options = { }; }
 
-    // Since jison allows \n, \r or \r\n line endings, we need some
```

Comment:
Jison is the application used to process the file (a JavaScript equivalent to Bison). ;)

Reply:
my bad :upside_down_face:

Reply:
No worries. I just finished going through your PR and merging it locally.

Thanks a lot for your attention to detail. :)